packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -62,6 +63,14 @@ class FlutterDevice {
          packagesPath: globalPackagesPath,
        );
 
+
+  static String getCachedDillPath({@required bool trackWidgetCreation}) {
```

Comment:
nit: I would make this a top level instead of a static, since it isn't really related to the FlutterDevice

Reply:
moved to `bundle.dart` as top level `getDefaultCachedKernelPath`